sdk/documenttranslator/ai-document-translator/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@azure-rest/ai-document-translator",
```

Comment:
So do we want to encode `-rest` into the folder path? Since this is being checked in under `documenttranslator/ai-document-translator`, if we later wanted to create an `@azure/ai-document-translator` package, I don't know where that would go.

Reply:
Yeah that makes sense I'll rename the folder